controllers/authpolicy_status.go

```diff
@@ -6,18 +6,16 @@ import (
 	"slices"
 
 	"github.com/go-logr/logr"
+	authorinoapi "github.com/kuadrant/authorino/api/v1beta2"
+	api "github.com/kuadrant/kuadrant-operator/api/v1beta2"
+	"github.com/kuadrant/kuadrant-operator/pkg/common"
```

Comment:
Do you mind separating these at least 2 imports to another block please?

Reply:
Sure 👍